Transmodel API changes

gmellemstrand opened the pull request:
This contains mostly changes to the transmodel sandbox api.

See changes below

abyrd reviewed (approved):
Although this is assigned to Thomas, I thought you might want to merge it before he gets back. I don't use the Transmodel API myself, but this changeset looks clean and only affects sandbox code so feel free to merge unless it's awaiting internal Entur approval.

gmellemstrand commented:
Thanks for the review, @abyrd . I forgot to mark this as WIP, as I am doing a throughout testing of the transmodel api, and I except to make more changes. Probably mostly sandbox changes, but you might want to look through them before you approve again.

gmellemstrand commented:
List of changes:

- Add intermediateEstimatedCalls to Leg
- Add fromEstimatedCall/toEstimatedCall to Leg
- Rename optimisationMethod to bicycleOptimasationMethod (also renamed graphql enum accordingly)
- Removed the following arguments from the trip call: allowBikeRental, vias, maximumWalkDistance, maxTransferWalkDistance, transportSubmodes, minimumTransferTime, compactLegsByReversedSearch, reverseOptimizeOnTheFly, maxPreTransitTime, maxPreTransitWalkDistance, useFlex
- Removed wheelChairBoarding and timeZone fields from StopPlace type
- Removed FlexibleArea and timeZone fields from the Quay type
- Make all StopPlace and Quay queries work correctly
- Prevent extra quotes in the BikeRentalStation ids by using the correct method to read the json string
- Set default maxTransfers to 12 and maxItineraries to 12

gmellemstrand commented:
There will be more changes, but they are a bit more extensive and require changes to the core OTP code, so I will make separate pull requests for those.

This should be ready for review now.

t2gran commented:
I added some documentation and changed the default value for numItineraries to 50.
